src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars--0 box">
+      <div class="stars normal-star" alt="Сіра зірка"></div>
+      <div class="stars normal-star" alt="Сіра зірка"></div>
```

Comment:
Don't hardcode these stars. You should give modifier to each block of stars (stars--0, stars--1, etc.) and depending on the modifier the number of active stars should change